src/components/Login/LoginScanCard.test.tsx

```diff
@@ -0,0 +1,27 @@
+import { render, fireEvent, cleanup } from "@testing-library/react-native";
+import React from "react";
+import { LoginScanCard } from "./LoginScanCard";
+import "../../common/i18n/i18nMock";
+
+const onToggleScanner = jest.fn();
+const isLoading = false;
+
+describe("LoginScanCard", () => {
+  afterEach(() => {
+    cleanup();
+    jest.resetAllMocks();
+  });
+
+  it("should be able to render without error", async () => {
+    expect.hasAssertions();
```

Comment:
We've been specifying the actual number of assertions so we make sure that all assertions are being asserted